frontend/src/concepts/projects/utils.ts

```diff
@@ -7,3 +10,12 @@ export const isAvailableProject = (projectName: string, dashboardNamespace: stri
     projectName === 'openshift' ||
     projectName === dashboardNamespace
   );
+
+export const getProjectDisplayName = (project: ProjectKind): string =>
+  getDisplayNameFromK8sResource(project);
+export const getProjectDescription = (project: ProjectKind): string =>
+  getDescriptionFromK8sResource(project);
```

Comment:
Not your fault this is just refactoring -- but these may be short lived. We started with not being able to get direct access to the base level `getDisplayNameFromK8sResource` -- and now it seems to have mixed usage. 

https://issues.redhat.com/browse/RHOAIENG-6359 will address this going forward.